src/internal/components/pluggableVisualizations/geoChart/PluggableGeoPushpinChart.tsx

```diff
@@ -0,0 +1,204 @@
+            },
+            {
+                localIdentifier: BucketNames.COLOR,
+                items: measuresColor,
+            },
+            {
+                localIdentifier: BucketNames.SEGMENT_BY,
+                items: segments,
+            },
+        ]);
+
+        newReferencePoint = setGeoPushpinUiConfig(newReferencePoint, this.intl, this.type);
+        newReferencePoint = getReferencePointWithSupportedProperties(
+            newReferencePoint,
+            this.supportedPropertiesList,
+        );
+        newReferencePoint = removeSort(newReferencePoint);
+
+        return Promise.resolve(sanitizeFilters(newReferencePoint));
+    }
+
+    protected renderConfigurationPanel() {
+        const configPanelElement = document.querySelector(this.configPanelElement);
+        if (configPanelElement) {
+            const properties: IVisualizationProperties = get(this, "visualizationProperties.properties", {});
+
+            render(
+                <UnsupportedConfigurationPanel
+                    locale={this.locale}
+                    pushData={this.callbacks.pushData}
+                    properties={properties}
+                />,
+                configPanelElement,
+            );
+        }
+    }
+
+    protected renderVisualization(
+        options: IVisProps,
+        visualizationProperties: IVisualizationProperties,
+        mdObject: VisualizationObject.IVisualizationObjectContent,
+    ) {
+        const { dataSource } = options;
+        if (dataSource) {
+            const {
+                dimensions: { height },
+                custom: { drillableItems },
+                locale,
+                config,
+            } = options;
+            const {
+                afterRender,
+                onDrill,
+                onFiredDrillEvent,
+                onError,
+                onExportReady,
+                onLoadingChanged,
+            } = this.callbacks;
+
+            // keep height undef for AD; causes indigo-visualizations to pick default 100%
+            const resultingHeight = this.environment === DASHBOARDS_ENVIRONMENT ? height : undefined;
+            const allProperties: IVisualizationProperties = get(
+                visualizationProperties,
+                "properties",
+                {},
+            ) as IVisualizationProperties;
+            const resultSpecWithDimensions: AFM.IResultSpec = {
+                ...options.resultSpec,
+                dimensions: this.getDimensions(mdObject),
+            };
+            const enableSortingByTotalGroup = this.featureFlags.enableSortingByTotalGroup as boolean;
+            const sorts: AFM.SortItem[] = createSorts(
+                this.type,
+                dataSource.getAfm(),
+                resultSpecWithDimensions,
+                allProperties,
+                canSortStackTotalValue(mdObject, null, enableSortingByTotalGroup),
+                enableSortingByTotalGroup,
+            );
```

Comment:
I don't think we need `sortStackTotalValue` and `enableSortingByTotalGroup` here. They should be false by default.
According to 3rd item in [this comment](https://jira.intgdc.com/browse/SD-780?focusedCommentId=1219684&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1219684), we should make Size measure always sorted DESC by default, so that smaller puspins are rendered later and on top of bigger ones.